waspc/data/Generator/templates/server/src/auth/utils.ts

```diff
@@ -32,7 +54,7 @@ export const authConfig = {
   successRedirectPath: "{= successRedirectPath =}",
 }
 
-export async function findAuthIdentity(providerName: string, providerUserId: string) {
+export async function findAuthIdentity(providerName: ProviderName, providerUserId: string) {
```

Comment:
Ok, we need to use `ProviderName` in other places too then, it is not used everywhere!

Reply:
:rocket: ?